tests/common/jwt/jwt_token_test.go

```diff
@@ -106,28 +107,13 @@ func TestSampleJwtToken(t *testing.T) {
 		}
 
 		for k, v := range tc.wantClaims {
-			switch v.(type) {
-			case string:
-				got, ok := claims[k].(string)
-				if ok {
-					if got != v {
-						t.Errorf("%s: claim %q got value %q but want %q", tc.name, k, got, v)
-					}
-				} else {
-					t.Errorf("%s: claim %q got type %T but want string", tc.name, k, claims[k])
+			got, ok := claims[k].(interface{})
```

Comment:
isn't `ok` here checking that the cast to `.(interface{})` succeeded? 

Also seems the cast is unneeded, isn't it already an `interface{}`?

Reply:
Ah, you're right, the cast is unneeded. The `ok` is checking if the key is present in the `claims`.

Reply:
Both a map lookup and cast return a second `ok` value. Looks like the end value is essentially that they both succeed https://play.golang.org/p/GijhZ5UT0jk. Learned something new today :slightly_smiling_face: